pages/en/developing/supported-networks.mdx

```diff
@@ -4,52 +4,27 @@ title: Supported Networks
 
 ## The Graph Decentralized Network
 
-The Graph Network supports subgraphs indexing the Ethereum mainnet and Rinkeby testnet:
+The following protocols are supported on The Graph Decentralized Network:
 
-- `mainnet`
-- `rinkeby`
+- Ethereum
+
+Indexers on The Graph Network can offer full support & security for mainnet Ethereum subgraphs. See which networks are supported and more information on deploying to the network [here](deploying/deploying-a-subgraph-to-studio).
+
+Remember that you will **not be able** to publish a subgraph that indexes a non-mainnet network to the decentralized Graph Network in [Subgraph Studio](/deploying/subgraph-studio).
 
 ## Hosted Service
 
-The following networks are supported in beta on the Hosted Service:
-
-- `mainnet`
-- `kovan`
-- `rinkeby`
-- `ropsten`
-- `goerli`
-- `poa-core`
-- `poa-sokol`
-- `xdai` (now known as Gnosis Chain)
-- `near-mainnet`
-- `near-testnet`
-- `matic` (now known as Polygon)
-- `mumbai`
-- `fantom`
-- `bsc` (now known as BNB Chain)
-- `chapel`
-- `clover`
-- `avalanche`
-- `fuji`
-- `celo`
-- `celo-alfajores`
-- `fuse`
-- `moonriver`
-- `mbase`
-- `arbitrum-one`
-- `arbitrum-rinkeby`
-- `optimism`
-- `optimism-kovan`
-- `aurora`
-- `aurora-testnet`
-- `boba`
-- `harmony`
-- `arweave-mainnet`
-- `cosmoshub-4`
-- `theta-testnet-001` (this is the current Cosmos Hub testnet)
+The following protocols are supported on the Hosted Service:
+
+- Ethereum
+- Near
+- Cosmos
+- Arweave 
+
+See the list of which networks are supported [here](deploying/deploying-a-subgraph-to-hosted).
 
 > The Graph's Hosted Service relies on the stability and reliability of the underlying technologies, namely the provided JSON RPC endpoints. Newer networks will be marked as being in beta until the network has proven itself in terms of stability, reliability, and scalability. During this beta period, there is a risk of downtime and unexpected behaviour.
 
-Remember that you will **not be able** to publish a subgraph that indexes a non-mainnet network to the decentralized Graph Network in [Subgraph Studio](/deploying/subgraph-studio).
+> Graph Node can index any EVM compatible chain.
```

Comment:
I think worth moving these notes to the top, and adding "Specific support depends where you are deploying your subgraph".

Reply:
Take a look! Hopefully, this is what you meant :) @azf20